cmd/hub/main.go

```diff
@@ -12,9 +13,21 @@ import (
 	"github.com/sonm-io/core/insonmnia/hub"
 )
 
+var (
+	configPath = flag.String("config", "hub.yaml", "Path to hub config file")
+)
+
 func main() {
+	flag.Parse()
 	ctx := context.Background()
-	h, err := hub.New(ctx)
+
+	conf, err := hub.NewConfig(*configPath)
+	if err != nil {
+		ctxlog.GetLogger(ctx).Fatal("Cannot load config", zap.Error(err))
```

Comment:
Hmm, not so explicit behaviour. Maybe rewrite using .Error() and then explicitly call os.Exit(1)?